awxkit/awxkit/cli/options.py

```diff
@@ -40,6 +40,11 @@ def pk_or_name(v2, model_name, value, page=None):
     if model_name in UNIQUENESS_RULES:
         identity = UNIQUENESS_RULES[model_name][-1]
 
+    # certain related fields follow a pattern of <foo>_<model> e.g.,
```

Comment:
This LGTM. I cross-referenced what this is doing with https://github.com/ansible/awx/blob/devel/awx/api/serializers.py#L93-L141 and I think this assumption would hold for anything on this list that qualifies.

I realize those are summary fields and not _necessarily_ what we might see when parsing out the options data to generate cli flags, but it's a decent-enough summary of many of the different fk field names we see in general.